modules/apps/item-selector/item-selector-web/src/main/resources/META-INF/resources/view_item_selector_view_descriptor.jsp

```diff
@@ -37,9 +37,11 @@ SearchContainer<Object> searchContainer = itemSelectorViewDescriptorRendererDisp
 		var="entriesSearch"
 	>
 		<liferay-ui:search-container-row
+			ariaLabel="select-the-item-and-close-the-modal"
 			className="Object"
 			keyProperty="<%= itemSelectorViewDescriptor.getKeyProperty() %>"
 			modelVar="entry"
+			tabindex="0"
```

Comment:
@pat270 FYI, you can use `itemSelectorViewDescriptorRendererDisplayContext.isMultipleSelection()` to conditionally add these attributes when it's not multipleSelection.